liquidity/threshold_rule.go

```diff
@@ -59,3 +61,58 @@ func (r *ThresholdRule) validate() error {
 
 	return nil
 }
+
+// loopOutSwapAmount determines whether we can perform a loop out swap, and
+// returns the amount we need to swap to reach the desired liquidity balance
+// specified by the incoming and outgoing thresholds.
+func loopOutSwapAmount(balances *balances, incomingThreshold,
```

Comment:
Maybe `minimumIncomingPerc` is a more consistent name

Reply:
Why `Perc`? It's an absolute amount, not a percentage. How about `requiredInboundAmount`? Or just leave it the way it is, I think `loopOutSwapAmount` also fits.

Reply:
It does look like a percentage to me. Below it is converted to an absolute amount?